src/app/open-forum/open-forum.component.css

```diff
@@ -0,0 +1,23 @@
+.aswwu-outline-dark {
```

Comment:
When the user hovers over the button, the outline disappears. Can we make it stay the whole time